CONTRIBUTING.md

```diff
@@ -232,6 +232,36 @@ the event happens.
 Make log message human readable and also include data that is needed for easier
 understanding of what happened and in what context.
 
+### Observability
+
+Out of the box, your users should be able to observe the state of your component. 
+The collector exposes an OpenMetrics endpoint at `/metrics` where your data will land.
```

Comment:
Is OpenMetrics the right term to use here or Prometheus? (I am not quite sure what the relation between the too is).

Reply:
Not a metrics expert, but my understanding is that OpenMetrics is the standard that defines the format in which those metrics are exposed.